package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kubric/featuregates",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
If we are to go with [semver rules](https://semver.org/), this more qualifies as a patch update as it is a minor bug fix and does not introduce any new features.

Reply:
i did think of that. But changed `x.Y.z` as it was a critical issue/feature.

i will bump the minor version